pkg/sink/codec/simple/message.go

```diff
@@ -431,7 +431,7 @@ func newDDLMessage(ddl *model.DDLEvent) *message {
 }
 
 func newDMLMessage(
-	event *model.RowChangedEvent, onlyHandleKey bool,
+	event *model.RowChangedEvent, config *common.Config, onlyHandleKey bool,
```

Comment:
Why did you include these changes in this pull request? It seems that they do not relate to the `schemaStorage`.